paddle/fluid/framework/details/ssa_graph_builder.cc

```diff
@@ -47,6 +47,17 @@ void SSAGraphBuilder::PolishGraphToSupportDataHazards(SSAGraph *graph) {
   }
 }
 
+VarHandle *SSAGraphBuilder::GetLatestVarHandle(SSAGraph *graph,
```

Comment:
Is this used some where?

Reply:
No, this function doesn't belong to shi PR.